GETTING_STARTED.md

```diff
@@ -0,0 +1,39 @@
+# getting_started
+
+Follow this guide to setup [Parrot Security](https://www.parrotsec.org/) with Docker on Oracle VirtualBox. Similar steps can be taken to setup [Kali Linux](https://www.kali.org/) instead.
```

Comment:
Can we give this a heading `## Parrot` and shift all subsequent headings by one level (one `#`). Under this heading could you give a brief introduction to Parrot and why it should be setup